### DISCUSSION TITLE
How to clear all notifications when app opened

### DISCUSSION THREAD

**Discussion by LakshanKarunathilake** [invertase/react-native-firebase#6071] [Q&A]
I want to clear all my firebase push notifications when the applications is opened, Is there a way to do it ?

**Comment by mikehardy** [comment]
Hi there! I'm not sure what you mean. What exactly is a "push notification"? I ask because people confuse visible notifications on devices (what are properly called "local notifications") and firebase cloud messages (FCM).

If you want to clear visible notifications on the device, you need a module that covers the API surfaces of local device notification APIs. That would be something like notifee.app

If you want to clear FCM, that's not something you are in control of, you either send messages or you don't, they deliver or they don't, but you cannot really "clear" them.

**Reply by LakshanKarunathilake** [reply]
Thank you very much for the quick response, got it I am trying to clear the FCM notifications. I was wondering whether there is a way to clear stuff without using notifee.

I'll use notifee in this case. 👌
